### PR TITLE
testing branchone

### PR DESCRIPTION
**Pull request by Serendipitously**
Left smiley faces where needed

### CODE REVIEW

**Comment by brisberg** [comment]
Time to code review

File: core/tests.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 from django.test import TestCase
 
 # Create your tests here.
+#hi :)
```
Comment (brisberg) [inline core/tests.py:4]: wat

Comment (Serendipitously) [inline core/tests.py:4]: smiley faces are important

**Comment by brisberg** [comment]
LGTM

**Pull request merged**